### PR TITLE
Update Website

### PR DESCRIPTION
**Pull request by lorenwel**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -199,33 +145,33 @@ <h3 class="h4 mb-2">Versatility</h3>
...
-                            <p class="text-muted mb-4">Large-scale reality capture that replicates real building into digital worlds!</p>
+                            <p class="text-muted mb-4">Large-scale reality capture of the real world into digital worlds!</p>
                         </div>
                     </div>
                     <div class="col-lg-3 col-md-6 text-center">
                         <div class="mt-1">
                             <div class="mb-2"><i class="fa-solid fa-brain fa-4x text-primary"></i></div>
                             <h3 class="h4 mb-2">Embodied AI</h3>
-                            <p class="text-muted mb-4">Reinforcement and supervised learning to enhance real-world reasoning and navigation skills!</p>
+                            <p class="text-muted mb-4">Reinforcement and supervised learning to connect AI with reality!</p>
```
Comment (lorenwel) [inline index.html:169]: reality --> real world

File: index.html [hunk 2]
```diff
@@ -199,33 +145,33 @@ <h3 class="h4 mb-2">Versatility</h3>
...
-                            <p class="text-muted mb-4">Our robot seamlessly transitions between a car, a four-legged robot, and a humanoid!</p>
+                            <p class="text-muted mb-4">Transitions between driving and walking on four and two legs!</p>
                         </div>
                     </div>
                     <div class="col-lg-3 col-md-6 text-center">
                         <div class="mt-1">
                             <div class="mb-2"><i class="fa-solid fa-earth-americas fa-4x text-primary"></i></div>
                             <h3 class="h4 mb-2">Digital Twins</h3>
-                            <p class="text-muted mb-4">Large-scale reality capture that replicates real building into digital worlds!</p>
+                            <p class="text-muted mb-4">Large-scale reality capture of the real world into digital worlds!</p>
```
Comment (lorenwel) [inline index.html:162]: Large-scale reality replicates the real world digitally!

File: index.html [hunk 3]
```diff
@@ -292,48 +238,54 @@ <h3 class="h4 mb-2">Embodied AI</h3>
...
                             <div class="mb-2"><i class="fa-solid fa-dolly fa-4x text-primary"></i></div>
                             <h3 class="h4 mb-2">Last-Mile Delivery</h3>
-                            <p class="text-muted mb-0">General-purpose platform for urban outdoor and indoor environments, allowing for various logistic applications by deploying goods, materials and tools.</p>
+                            <p class="text-muted mb-4">General-purpose platform for urban environments, allowing various logistic applications by carrying goods, materials and tools.</p>
                         </div>
                     </div>
                     <div class="col-lg-3 col-md-6 text-center">
                         <div class="mt-5">
                             <div class="mb-2"><i class="fa-solid fa-video fa-4x text-primary"></i></div>
                             <h3 class="h4 mb-2">Industrial Monitoring</h3>
```
Comment (lorenwel) [inline index.html:255]: Consider removing this until we have cleared everything next Monday.

File: index.html [hunk 4]
```diff
@@ -292,48 +238,54 @@ <h3 class="h4 mb-2">Embodied AI</h3>
...
         <!-- Projects-->
-        <section class="page-section bg-dark text-white" id="projects">
+        <section class="page-section bg-dark text-white" id="tech">
             <div class="container px-4 px-lg-5">
                 <div class="row gx-4 gx-lg-5 justify-content-center">
                     <div class="col-lg-8 text-center">
-                        <h2 class="text-white mt-0">Projects</h2>
+                        <h2 class="text-white mt-0">Technology</h2>
                         <hr class="divider divider-light" />
+                        <p class="text-white-75 mb-4">Within the Robotic Systems Lab of ETH Zurich, the founders have dedicated six years to pioneering research in robotic intelligence and wheeled-legged robots. The founders built on a third-party robot and further customized it strictly for research purposes.</p>
```
Comment (lorenwel) [inline index.html:288]: Remove the "have" here: `Within the Robotic Systems Lab of ETH Zurich, the founders have dedicated six years to pioneering research in robotic intelligence and wheeled-legged robots`

**Review by areske — APPROVED** [review]

**Review by mbjelonic — APPROVED** [review]

**Pull request merged**